### ISSUE TITLE
New app producing issue: return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]))

### ISSUE THREAD

**Issue by flexdhana** [facebook/react-native#10458]
I created new app using react-native init, and I am getting the following issue in main.m when i run as it is, please suggest..

return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
- React Native version: 0.35
- Platform(s) (iOS, Android, or both?): iOS
- XCODE: 7.3.1

<img width="857" alt="screen shot 2016-10-19 at 2 41 34 pm" src="https://cloud.githubusercontent.com/assets/19851520/19512563/3176d04a-960a-11e6-8169-3152075f01ea.png">

**Comment by flexdhana** [comment]
Following solution solves the issue (temporarily)

By Changing Product > Scheme > Edit Scheme > set build configuration to Release.

http://stackoverflow.com/questions/38780325/running-react-native-app-on-ios-device-using-offline-bundle

**Comment by lacker** [comment]
Hmm, can you explain your situation a bit more? In particular, what are you doing that leads to an error - you aren't just running `react-native init` and starting the app and it fails, is it? Also what about that Stack Overflow solution isn't good enough?

**Comment by CapitanRedBeard** [comment]
This might be too specific for others, but I came across this error as well and here was the situation.

Newly detached expo app, messing with manually linking files, eventually turned off parallelize build and added React to my target in Product > Scheme > Edit Scheme > Build.

After fixing up my links I started getting this error and all I had to do to resolve it was go back and enable parallelize build again and remove the React target from the Scheme.

Cheers 🍻
